helpers.lua

```diff
@@ -170,6 +170,27 @@ function helpers.spairs(t)
     end
 end
 
+-- create trivial partition of a set
+function helpers.trivial_partition_set(set)
```

Comment:
I'd rather call it `partition_of_singletons` or add the descriptions from the commit message to the code.